docs/package-manager.org

```diff
@@ -1,25 +1,27 @@
-* Favored package manager:
+* Favored package manager
 
 Recently we dropped the support for the ~npm~.
 The decision was made upon the several problems we came across while using this package manager.
 
-[[https://zaufanatrzeciastrona.pl/post/menadzer-paczek-node-npm-brickuje-systemy/][NPM brickuje systemy]]
-[[https://www.theinquirer.net/inquirer/news/3027293/bug-found-in-popular-javascript-package-manager-nmp-update-causes-system-crashes][NPM update causes system crashes]]
+- [[https://zaufanatrzeciastrona.pl/post/menadzer-paczek-node-npm-brickuje-systemy/][NPM brickuje systemy]]
```

Comment:
Please add PL and EN langauge hint
Example: NPM brickuje systemy *(PL)*

Reply:
Thanks added :)